src/Reporting/ReportRow.php

```diff
@@ -71,15 +71,17 @@ public function next() {
 		 */
 		public function key() {
 			if ($this->keys == null)
-				$this->keys = array_keys($this->data);
+				$this->keys = array_keys((array)$this->data);

```

Comment:
This should probably be get_object_vars(), generally speaking casting objects to arrays are not that great